lxd/storage/backend_lxd.go

```diff
@@ -2464,14 +2474,14 @@ func (b *lxdBackend) CreateCustomVolumeFromMigration(projectName string, conn io
 	volStorageName := project.StorageVolume(projectName, args.Name)
 
 	// Check the supplied config and remove any fields not relevant for destination pool type.
-	vol := b.newVolume(drivers.VolumeTypeCustom, drivers.ContentTypeFS, volStorageName, args.Config)
+	vol := b.newVolume(drivers.VolumeTypeCustom, drivers.ContentType(args.ContentType), volStorageName, args.Config)
```

Comment:
@monstermunchkin hopefully will be able remove this conversion by using the native storageDrivers.ContentType in the struct.